deployment/openshift/about.md

```diff
@@ -1,12 +1,168 @@
+* Web console and command-line client for building and monitoring applications.
+* Centralized administration and management of an entire stack, team, or organization.
+  * Create reusable templates for components of your system, and iteratively deploy them over time.
+  * Roll out modifications to software stacks to your entire organization in a controlled fashion.
+  * Integration with your existing authentication mechanisms, including LDAP, Active Directory, and public OAuth providers such as GitHub.
+* Multi-tenancy support, including team and user isolation of containers, builds, and network communication.
+  * Allow developers to run containers securely with fine-grained controls in production.
+  * Limit, track, and manage the developers and teams on the platform.
+* Integrated Docker registry, automatic edge load balancing, cluster logging, and integrated metrics.
+
+## Start a local OpenShift all-in-one cluster
+
+The quickest way to try out OpenShift on your own computer is using the `oc cluster up` command.
+First, install the dependencies. We'll need [Docker](/tools/docker/docker-installation.html) and the `origin-clients` that provides the `oc` binary.
+
+```console
+$ sudo dnf install origin-clients docker
+```
+
+*Note*: we recommend following the steps in the [Docker installation page](/tools/docker/docker-installation.html#why-cant-i-use-docker-command-as-a-non-root-user-by-default) to add your user to the `docker` group, so that you can run the `docker` command without `sudo`.
+
+Now, let's start our local all-in-one cluster. The command `oc cluster up` takes care of everything:
+
+```console
+$ oc cluster up
+-- Checking OpenShift client ... OK
+-- Checking Docker client ... OK
+-- Checking Docker version ... OK
+-- Checking for existing OpenShift container ... OK
+-- Checking for openshift/origin:v1.3.1 image ...
+   Pulling image openshift/origin:v1.3.1
+   Pulled 0/3 layers, 3% complete
+   Pulled 1/3 layers, 71% complete
+   Pulled 2/3 layers, 86% complete
+   Pulled 3/3 layers, 100% complete
+   Extracting
+   Image pull complete
+-- Checking Docker daemon configuration ... FAIL
+   Error: did not detect an --insecure-registry argument on the Docker daemon
+   Solution:
+
+     Ensure that the Docker daemon is running with the following argument:
+        --insecure-registry 172.30.0.0/16
+```
+
+If you are running a default installation of Docker, you will run into the error above.
+That is okay. We will follow the suggestion on screen. The good news is that `oc cluster up` downloaded the Docker image for running OpenShift Origin, so next time we run it, we already have the images offline.
+
+To change the Docker configuration permanently, edit (as the `root` user) the file `/etc/sysconfig/docker` and add/uncomment the following line:
+
+```
+INSECURE_REGISTRY='--insecure-registry 172.30.0.0/16'
+```
+
+Then, restart the Docker daemon:
+
+```console
+$ sudo systemctl restart docker
+```
```

Comment:
It was an intentional "mistake", part of the didactic method. https://coach.djangogirls.org/tips/index.html#learn-from-mistakes

Yes we can guide people towards seemingly perfection, but I decided here to show how to interpret and react to error messages, they are not so uncommon and unlike to happen.

In fact, I had the missing insecure registry config when I was testing these steps in a VM.